suite-common/wallet-utils/src/solanaStakingUtils.ts

```diff
@@ -0,0 +1,71 @@
+import { Solana, SolNetwork } from '@everstake/wallet-sdk';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { BigNumber, isArrayMember } from '@trezor/utils';
+import { SolanaStakingAccount } from '@trezor/blockchain-link-types/src/solana';
+import {
+    Account,
+    supportedSolanaNetworkSymbols,
+    SupportedSolanaNetworkSymbols,
+} from '@suite-common/wallet-types';
+import { LAMPORTS_PER_SOL } from '@suite-common/wallet-constants';
+
+export function isSupportedSolStakingNetworkSymbol(
+    networkSymbol: NetworkSymbol,
```

Comment:
Use please `symbol` instead of `networkSymbol`

Reply:
Is this important? This is not for display but actually based on network

Reply:
Or you know what, I'll just change it 😃

Reply:
Gracia 🫶